client/src/components/Cities.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {getCities} from '../actions/citiesActions';
+import {connect} from 'react-redux'; 
+
+class Cities extends Component {
+    constructor() {
+      super();
+      this.state = {
+        search: '',
+        //cities:[]
+      };
+    }
+
+    updateSearch(event) {
+      this.setState({search: event.target.value});
+    }
+
+  componentDidMount() {
+    // axios.get("/api/city").then(res => {
+    // console.log(res);
+    //   this.setState({ cities: res.data });
+      this.props.getCities();
+    }
+  
+
+  render() {
+    console.log(this.props)
+      let filteredCities = this.props.cities.cities.filter(
+        (city) => {
+          return city.name.toLowerCase().indexOf(this.state.search) !==-1;
+        }
+      );
+      return (
+      <div>
```

Comment:
I think this DOM structure could be refactored to use more semantic elements or to include meaningful classes.  It's confusing to understand what this outputs by looking at the template.